.github/workflows/test.yaml

```diff
@@ -47,6 +51,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v1
+        with:
+          ref: feat_last_attempt
```

Comment:
Remove all instances of this ref before the merge, so we are checking out master when in master.